crates/swift-bridge-ir/src/codegen/codegen_tests/opaque_swift_type_codegen_tests.rs

```diff
@@ -65,3 +65,121 @@ func __swift_bridge__some_function (_ arg: UnsafeMutableRawPointer) {
         .test();
     }
 }
+
```

Comment:
Moved here from `extern_swift_function_opaque_swift_type_return_codegen_tests.rs`... the original issue mentioned moving those tests